### ISSUE TITLE
Too damn many unconfirmed bugs

### ISSUE THREAD

**Issue by appgurueu** [minetest/minetest#11998]
Title says it [all](https://github.com/minetest/minetest/issues?q=is%3Aissue+is%3Aopen+label%3A%22Unconfirmed+bug%22). Many of these should long have been retagged to "Bug".

**Comment by Zughy** [comment]
#11568 might help

**Comment by grorp** [comment]
Really like this "Not a bug" label :D

**Comment by Wuzzy2** [comment]
Do you want "label rights"? If yes, then say so. Maybe you get them.

**Comment by appgurueu** [comment]
> Do you want "label rights"? If yes, then say so. Maybe you get them.

I just want the issues to be labeled correctly, that's all - I don't care how they get there.

**Comment by NathanSalapat** [comment]
<sarcasm>Well maybe if you confirmed every unconfirmed bug they could be changed.</sarcasm>

Seriously though, if no core dev is going around checking every bug report to see if it's valid there's not much that can be done about it.

**Comment by appgurueu** [comment]
> [sarcasm]Well maybe if you confirmed every unconfirmed bug they could be changed.[/sarcasm]
> 
> Seriously though, if no core dev is going around checking every bug report to see if it's valid there's not much that can be done about it.

Let me rephrase: I believe the bar for "bug confirmation" is way too high. The default is that bugs are assumed to be unconfirmed - fair enough. But if a detailed report is provided, that alone should count as "confirmation".

**Comment by appgurueu** [comment]
> [sarcasm]Well maybe if you confirmed every unconfirmed bug they could be changed.[/sarcasm]
> 
> Seriously though, if no core dev is going around checking every bug report to see if it's valid there's not much that can be done about it.

Let me rephrase: I believe the bar for "bug confirmation" is way too high. The default is that bugs are assumed to be unconfirmed - fair enough. But if a detailed report is provided, that alone should count as "confirmation".

**Comment by randomMesh** [comment]
I like how this issue it both an unconfirmed bug but at the same time is not a bug. You guys need to get your labels straight.

**Comment by Zughy** [comment]
I suggest to close this issue, I don't know what could be the purpose of it, except underlying a meta problem (maybe more suited for the forum)
